tailwind.config.js

```diff
@@ -133,6 +133,9 @@ module.exports = {
         '3xl': '1920px',
         print: {raw: 'print'},
       },
+      width: {
```

Comment:
opinion: seeing tailwind global config updated to affect change in (what I'm assuming is) a one-off problem is concerning and makes me think that perhaps we should sometimes "just use css" if tailwind's config isn't doing the work we need it to do